app/src/main/java/org/literacyapp/authentication/helper/DetectionHelper.java

```diff
@@ -154,6 +153,20 @@ private static synchronized void setScreenBrightness(Context context, double cur
         }
     }
 
+    private static synchronized void setDisplayTemperatureNight(){
+        try {
+            int displayTemperatureNight = getDisplayTemperatureNight();
+            if (displayTemperatureNight <= DISPLAY_TEMPERATURE_NIGHT_DEFAULT){
```

Comment:
you're right..the problem was, that I had only one variable in the beginning but then changed some things